src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/connection/ConfluentRestCalls.java

```diff
@@ -0,0 +1,149 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright © 2021 Atruvia AG <opensource@atruvia.de> as contributor to the ODPi Egeria project. */
+package org.odpi.openmetadata.adapters.connectors.integration.eventschema.connection;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.ssl.TrustStrategy;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestTemplate;
+
+import javax.net.ssl.SSLContext;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.X509Certificate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * This class implements the connection to a real Atlas System
+ */
+public class ConfluentRestCalls implements ConnectionStrategy {
+
+    /*
+     * This fields have to be set by egeria/ the atlas connector.
+     * First we concentrate on the functionality.
+     */
+    final private String targetURL;
+
+    public ConfluentRestCalls(String targetURL) {
+        super();
+        this.targetURL = targetURL;
+    }
+
+    /**
+     * REST call to retrieve a list of all subjects.
+     *
+     * @return List of all subjects, if no subjects are listed, an empty array is returned.
+     */
+    @Override
+    public List<String> listAllSubjects() {
+        RestTemplate restTemplate;
+        try {
+            restTemplate = restTemplate();
+        } catch (Exception e) {
+            //TODO: Exception handling
```

Comment:
maybe create an issue for this and the other exception processing  to track separately